tools/cockpit.spec

```diff
@@ -81,6 +81,13 @@ Source0:        https://github.com/cockpit-project/cockpit/releases/download/%{v
 %define build_optional 1
 %endif
 
+# Ship custom SELinux policy only in Fedora and RHEL-9 onward
+%if 0%{?rhel} >= 9 || 0%{?fedora}
+%define selinuxtype targeted
+%define with_selinux 1
+%define selinux_policy_version %(rpm --quiet -q selinux-policy && rpm -q --queryformat "%{V}" selinux-policy || echo 1)
```

Comment:
Just in case you have to push again, please add a comment that the fallback is for packit.

Reply:
Just realised that this does not contain release number. The version  can easily stay the same for 1/2 a year in this case. Please use both version and release (as is the case in %selinux_requires macro).